blogs/2022/09/15/dev-container-features.md

```diff
@@ -0,0 +1,139 @@
+---
+Order: 76
+TOCTitle: Dev Container Features
+PageTitle: Dev Container Features
+MetaDescription: What are dev container Features and what's new
+Date: 2022-09-15
+Author: Brigit Murtaugh
+---
+
+# Dev Container Features
+
+September 15, 2022 by Brigit Murtaugh, [@BrigitMurtaugh](https://twitter.com/BrigitMurtaugh)
+
+We’ve all had that moment when setting up our development environment – "Oh, I just need one more thing!" – that "thing" being one more language or toolset (or maybe a few more 😊) to work on your project.
+
+Development containers are a great way to simplify environment set up - they provide a full-featured coding environment with the tools your project needs. They’re configured using an image, Dockerfile, or Docker Compose file and `devcontainer.json`, which is a metadata format used to enrich containers with development specific content and settings.
+
+When creating a dev container, you may encounter the same "I just need one more thing!" – maybe you’re using a Node.js image in your Dockerfile and just need to add Git. Or maybe you need to add something more complex, like working with Docker or Kubernetes from within your dev container. Dev containers are great since anyone accessing your code will have the same, consistent experience with all those tools you added – but what’s the best way to add them?
```

Comment:
Possible word tweak? The original sentence seemed to hang a bit.

```suggestion
When creating dev containers, you may have the same "I just need one more thing!" reaction repeatedly – maybe you’re using a Node.js image in your Dockerfile and just need to add Git. Or maybe you need to add something more complex, like working with Docker or Kubernetes from within your dev container. Dev containers are great since anyone accessing your code will have the same, consistent experience with all those tools you added – but what’s the best way to add them?
```